include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -399,6 +399,24 @@ typedef struct OrtCUDAProviderOptions {
   */
   OrtArenaCfg* default_memory_arena_cfg;
 
+  /** \brief External function to allocate GPU memory. It replaces ORT's
```

Comment:
OrtCUDAProviderOptions has been deprecated in favor of the opaque OrtCUDAProviderOptionsV2

Reply:
I see. I will remove changes for `OrtCUDAProviderOptions`.